google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -1972,9 +1972,18 @@ def load files, format: nil, create: nil, write: nil,
         # need to complete a load operation before the data can appear in query
         # results.
         #
+        # Because BigQuery's streaming API is designed for high insertion rates,
```

Comment:
@quartzmo This text was taken directly from the troubleshooting guide. What are your thoughts on this? Can/should this be more clear?

Reply:
It seems good to me except for this phrase "the underlying table metadata exhibit are eventually". Can we fix that (everywhere) by just removing "exhibit"?

Reply:
Updated.

Reply:
@quartzmo Ping.